proposals/simd/TextSIMD.md

```diff
@@ -20,8 +20,8 @@ The canonical text format used for printing `v128.const` instructions is
 v128.const i32x4 0xNNNNNNNN 0xNNNNNNNN 0xNNNNNNNN 0xNNNNNNNN
 ```
 
-### v8x16.shuffle
+### v8x16.shuffle2_imm
 
 ```
-v8x16.shuffle i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5
+v8x16.shuffle2_imm i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5 i5
```

Comment:
This is a bit confusing. These are not really `i5`s, but they are `i8`s restricted to the `i5` domain.

Reply:
Maybe this should use `LaneIdx32` ?

Reply:
This is specifying text format so I believe i5 is accurate.